src/engine.js

```diff
@@ -147,6 +147,7 @@
 			{
 				if (e.files[0].type == "audio/mp3"
 					|| e.files[0].type == "audio/wave"
+				    	|| e.files[0].type == "audio/wav"

```

Comment:
I have found some more (which I am using in my own app), not sure it it really make sense, but just to mention it:
     'audio/vnd.wave' /*wav*/,
     'audio/x-wav' /*wav*/,
and also
    'audio/x-aiff' /*aiff*/,